catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/event/SendEvent.java

```diff
@@ -132,7 +154,9 @@ public SendEvent(TransformerManager transformerManager, GetRecordsType request,
         this.resultType =
                 request.getResultType() == null ? ResultType.HITS : request.getResultType();
 
-        List providers = ImmutableList.builder().add(new CswRecordCollectionMessageBodyWriter(transformerManager)).build();
+        List providers = ImmutableList.builder()
```

Comment:
Try `ImmutableList.of(new CswRecordCollectionMessageBodyWriter(transformerManager))`